backend/index.py

```diff
@@ -171,6 +176,14 @@ def run_spider(spider_name):
         with clients_lock:
             for ws in clients:
                 ws.send(f"Error running spider {spider_name}: {e}")
+def wipe_data():
+    logging.info("Wiping the database data")
+    meta = db.metadata
+    for table in reversed(meta.sorted_tables):
+        logging.info(f"Deleting data from table {table.name}")
+        db.session.execute(table.delete())
```

Comment:
Does this drop the table or just delete the data?

Reply:
It should just delete the data rather than drop the entire table